Adds a small medbay to the brig.

matskuman5 opened the pull request:
#1952 

![image](https://cloud.githubusercontent.com/assets/11191939/23813277/c90b5ae2-05e6-11e7-94f2-5432c624f933.png)

Replaces the old interrogation observation which was never used.

The defib is taken from the insanity ward.

:cl:
rscdel: The old interrogation observation room has been bulldozed...
rscadd: ...and replaced with a new, small medbay for the brig! Contains a sleeper and some first aid kits, as well as a syringe gun and morphine.
/:cl:

Super3222 commented:
I liked the interrogation room as a security officer, but I'll take what I get it. :ok_hand:

matskuman5 commented:
>I liked the interrogation room as a security officer, but I'll take what I get it. :ok_hand:

This replaces interrogation OBSERVATION, not the actual interrogation room

Super3222 commented:
I meant the observation room. It was fun eavesdropping on detective's in work.

Super3222 commented:
I meant the observation room. It was fun eavesdropping on detective's.

Lightning38088 commented:
Will the viro security suit be placed in there as well?

Time-Green commented:
Sleepers are acces locked to medical ID's. Did you change the acces on it?

Lightning38088 commented:
also adding a pair of handcuffs or straitjacket would be nice.

AlpacaGoesCrazy commented:
Interogation observation was 'a subtle place noone checks' in the brig. Best for hiding and ambushing security stuff. And now you cant do it anymore especially with a transparent airlock. Also a sleeper makes brig way more independant then it should be. (Maybe sleeper assembly would be better, requiring science to finish it?)

matskuman5 commented:
>especially with a transparent airlock.

The airlock is the same as ever. DM just shows it as that.

>Will the viro security suit be placed in there as well?

Yes.

>Sleepers are acces locked to medical ID's. Did you change the acces on it?

Uhh...

Not sure how to do that

Lightning38088 commented:
"a little more independent" is a lot better than just watching security members bring in critted people and letting them die only to go to medbay 10 mins later wasting the players time.

Time-Green commented:
I cant do it now, but I am pretty sure it has a variable saying wich acces it needs, change that to whatever the one for the brig is.

matskuman5 commented:
Alright I removed all the other medical stuff from the brig. I also added a normal medkit to the medical room.

Super3222 commented:
You need to remove the syringe gun from the brig, Mat. Otherwise we'll get officers GG'ing antagonists with syringe guns filled with morphine. Which is terrible because it's going to lead into the power creep of taking down criminals and antagonists with X instead of Y.

Cruix commented:
Also you should add a little cargo shuttle so they don't have to go to cargo for implants.

Frostben commented:
wait why not merge this with the morgue in perma instead? Wouldn't that be a better use of space and whatnot?

Super3222 commented:
That's an insanity ward for containing "crazy" prisoners. It wouldn't make sense to merge the two together.

AdamElTablawy commented:
Yeah, let's remove medbay altogether and give every department their own medbay, for quality of life.

Get rid of the sleeper.

matskuman5 commented:
They still have no way to heal from crit.

This is mostly to prevent the 100s of prisoners from bleeding to death in their cells because "lol im busy and medbay is too far"

matskuman5 commented:
I will remove the syringe gun though.

Super3222 commented:
To be fair, all you need is a sleeper to completely heal up. Other than that prisoners will be taken to medbay for blood transfusions.

As a suggestion, maybe you could replace the sleeper with another table with a box of body bags over it.

matskuman5 commented:
>To be fair, all you need is a sleeper to completely heal up.

AFAIK, sleepers can't heal from crit can they?

If I'm wrong I'm up to remove them

Mrcreepssss commented:
i think there's already a small medbay near isolation cell/ execution room?

AdamElTablawy commented:
Sleepers can heal you from crit. Epiphenerine will stabilize and slowly heal until at a point where the other chems can be injected.

So no.

Also if a prisoner is dying you take them to medbay. End of discussion. No self sustainability for shitcurity.

matskuman5 commented:
I removed the sleeper and added some bodybags.

Also added a reinforced window so it looks nice :3

![image](https://cloud.githubusercontent.com/assets/11191939/24018355/40341ed8-0a9c-11e7-96f6-6f090facc370.png)

matskuman5 commented:
17 days pls

Super3222 commented:
@Cruix Can we please get a final decision?

Cruix commented:
Final decision from me is no.

I would not object to the brig having some brute medkits, but a sleeper is infinite and just makes the brig an even more unassailable base. If you really want a sleeper, you can have one built there by science or have the CMO deconstruct one of the medbay ones.

matskuman5 commented:
@cruix I removed the sleeper, if you scroll 10 cm upwards youd see

doerakpoes commented:
Is it a common thing for coder to miss things when reading?
isn't reading well part of coding?

Cruix commented:
You are welcome to look through the thousands of PRs I have read, made, closed, and merged on three repositories and bring back statistics.

matskuman5 commented:
Ooh, what's this button do

matskuman5 commented:
So uh... @Cruix what's the final verdict

oisin100 commented:
wha-

matskuman5 commented:
27 days pls

Cruix commented:
Are you going to fix #2097 first or keep asking

matskuman5 commented:
Fixed @Cruix
